Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -69,7 +69,7 @@
         "repositoryURL": "https://github.com/skhamis/rust-components-swift",
         "state": {
           "branch": "update-for-fxios",
-          "revision": "1f1aba21157b0da730a5f3c7029bfca448cbec5e",
+          "revision": "c8500c46d0cced489ede2dc3573c188705f199b6",
```

Comment:
There is an error about that commit not being found in the repo... 
`could not find the commit 1f1aba21157b0da730a5f3c7029bfca448cbec5e in https://github.com/skhamis/rust-components-swiftxcodebuild: error: Could not resolve package dependencies:`

I could not find it: https://github.com/mozilla/rust-components-swift/commit/1f1aba21157b0da730a5f3c7029bfca448cbec5e